drivers/base/bootvid/i386/xbox/bootvid.c

```diff
@@ -105,139 +105,6 @@ ApplyPalette(VOID)
     }
 }
 
-/* PUBLIC FUNCTIONS **********************************************************/
```

Comment:
What's the purpose of the functions reordering? Can we strip that (unnecessary) commit from the otherwise nice PR?

Reply:
yes that's actually what I'm going to do :+1: